resources/views/script.blade.php

```diff
@@ -4,44 +4,158 @@
     <!-- [ELEVAR] Start -->
     <script id="elevar-dl-listener-config" type="application/json">
         {
-            "data_layer_listener_script": "https://shopify-gtm-suite.getelevar.com/shops/{{ $settings->uuid }}/events.js",
+            "data_layer_listener_script": "https://shopify-gtm-suite.getelevar.com/shops/{{ $settings->uuid }}/3.2.10/events.js",
             "ss_url": {!! blank($settings->server_side_url) ? 'null' : '"' . $settings->server_side_url . '"' !!}
+            "signing_key": {{ $settings->signing_key }},
+  	        "myshopify_domain": "{{ Str::of(settings(Astrogoat\Shopify\Settings\ShopifySettings::class)->url)->replace('https://', '') }}"
```

Comment:
Can we make the Shopify app a dependency of this package in the `composer.json` file if we now need it here?